en/configure-a-tidb-cluster.md

```diff
@@ -479,6 +479,23 @@ spec:
 >
 > If the TiKV version is earlier than 4.0.14 or 5.0.3, due to [a bug of TiKV](https://github.com/tikv/tikv/pull/10364), you need to configure the timeout `spec.tikv.evictLeaderTimeout` as large as possible to ensure that all Region leaders on the TiKV Pod can be evicted within the timeout. If you are not sure about the proper value, greater than '1500m' is recommended.
 
+### Configure graceful upgrade for TiCDC cluster
+
```

Comment:
```suggestion

> **Note:**
>
> - If the TiCDC version is earlier than v6.3.0, TiDB Operator forces an upgrade on TiCDC, which might cause replication latency increase.
> - The feature is available since TiDB Operator v1.4.0.

```